resources/scss/componets/06_header.scss

```diff
@@ -0,0 +1,208 @@
+// preloader start
+
+#preloader {
+    position: fixed;
+    z-index: 99999999;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $black;
+
+    .loader, .loader:before, .loader:after {
+        border-radius: 50%;
+        width: 2.5em;
+        height: 2.5em;
+        animation-fill-mode: both;
+        animation: bblFadInOut 1.8s infinite ease-in-out;
+    }
+
+    .loader {
+        color: #FFF;
+        font-size: 7px;
+        position: relative;
+        text-indent: -9999em;
+        transform: translateZ(0);
+        animation-delay: -0.16s;
+    }
+
+    .loader:before,
+    .loader:after {
+        content: '';
+        position: absolute;
+        top: 0;
+    }
+
+    .loader:before {
+        left: -3.5em;
+        animation-delay: -0.32s;
+    }
+
+    .loader:after {
+        left: 3.5em;
+    }
+
+    @keyframes bblFadInOut {
+        0%, 80%, 100% {
+            box-shadow: 0 2.5em 0 -1.3em
+        }
+        40% {
+            box-shadow: 0 2.5em 0 0
+        }
+    }
+}
+
+// preloader end
```

Comment:
Move this scss into **basics.scss** and remove this from this file please

Reply:
i move this preloader code into 01_basic.scss